docs/conf.py

```diff
@@ -15,18 +15,32 @@
 # import os
 # import sys
 # sys.path.insert(0, '/usr/local/src/libioc/iocage')
-
+import sphinx.apidoc
+import os.path
+
+def setup(app):
+    sphinx.ext.apidoc.main([
+        '--separate',
+        '-H', 'FOOOO',
+        '--no-toc',
+        '-o', 'docs',
+        '--full',
+        'ioc',
+        'tests'
+    ])
 
 # -- Project information -----------------------------------------------------
 
 project = 'libioc'
-copyright = '2018, ioc Contributors'
+copyright = '2019, ioc Contributors'
 author = 'ioc Contributors'
```

Comment:
it's funny that the docs are © ioc Contributors, but all the code is © Stefan Grönke

Reply:
Changed it to automatically obtain the copyright info from `LICENSE.txt`. This automation step eases keeping all information current.